.conda-envs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+package:
+  name: "pyatlan"
+  version: "{{ environ['GIT_DESCRIBE_TAG'] }}"
+
+source:
+  path: ../../
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pip
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
+  run:
+    - jinja2 ==3.1.4
+    - lazy_loader~ 0.4
+    - pydantic >=2.0.0,<3.0.0
+    - python
+    - requests >=2.24
+    - tenacity ==8.2.3
+    - urllib3 >=1.26.0,<3
+
+test:
+  imports:
+    - pyatlan
+    - pyatlan.cache
+    - pyatlan.client
+    - pyatlan.events
+    - pyatlan.generator
+    - pyatlan.generator.templates
+    - pyatlan.model
+    - pyatlan.model.assets
+    - pyatlan.model.assets.core
+    - pyatlan.model.fields
+    - pyatlan.model.packages
+    - pyatlan.model.packages.base
+    - pyatlan.pkg
+    - pyatlan.pkg.templates
+    - tests
+    - tests.integration
+    - tests.unit
+    - tests.unit.model
+    - tests.unit.model.fields
+    - tests.unit.pkg
+
+about:
+  home: "https://github.com/atlanhq/atlan-python"
+  license: Apache Software
+  license_family: APACHE
+  license_file:
```

Comment:
We can add these as well:

`license_file: LICENSE`
`license_url: https://github.com/atlanhq/atlan-python/blob/main/LICENSE`

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#about-section

Reply:
done